api/tests/views_tests.py

```diff
@@ -201,3 +214,166 @@ def test_post_relayaddress_flagged_error(free_user, free_api_client) -> None:
         "detail": "Your account is on pause.",
         "error_code": "account_is_paused",
     }
+
+
+class TermsAcceptedUserViewTest(TestCase):
+    def setUp(self):
+        self.factory = RequestFactory()
+        self.path = "/api/v1/terms-accepted-user/"
+        self.fxa_verify_path = authentication.INTROSPECT_TOKEN_URL
```

Comment:
note: if you accept the above suggestion, this would change too:
```suggestion
        self.fxa_verify_path = INTROSPECT_TOKEN_URL
```

Reply:
Resolved 0c7673f